Source/SIMPLib/CoreFilters/ImportAsciDataArray.cpp

```diff
@@ -428,14 +428,14 @@ void ImportAsciDataArray::dataCheck()
     return;
   }
 
-  QVector<size_t> tDims = attrMat->getTupleDimensions();
+  std::vector<size_t> tDims = attrMat->getTupleDimensions();
   size_t totalDim = 1;
   for(int i = 0; i < tDims.size(); i++)
```

Comment:
Side note: this should give a signed / unsigned conversion warning now.  `size()` really should (and now will) return an unsigned value.